### PR TITLE
add readme doc

### PR DESCRIPTION
**Pull request by maz-e**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# HELLO WORLD
+
+## Intro
```
Comment (edfrias) [inline README.md]: @maz-e intro no, but introduction yes

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+TODO explain project
```
Comment (edfrias) [inline README.md]: @maz-e you should finish this TODO

File: README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## Functional Description
+
+TODO explain functionality
```
Comment (edfrias) [inline README.md]: @maz-e hey you should complete this TODO

File: README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+## Technical Description
+
+TODO describe specs
```
Comment (edfrias) [inline README.md]: @maz-e hey you should complete this TODO

File: README.md (hunk 1 above)
Comment (maz-e) [reply README.md]: @edfrias Nop, is correct!

File: README.md (hunk 2 above)
Comment (maz-e) [reply README.md]: @edfrias  Arreglado!

File: README.md (hunk 3 above)
Comment (maz-e) [reply README.md]: @edfrias  Arreglado!

**Pull request merged**